<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DomainInfoWidget</class>
 <widget class="QWidget" name="DomainInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_UrlStatus">
        <property name="title">
         <string>Url status</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_UrlTotal">
           <property name="text">
            <string>Total</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_UrlTotal</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_UrlTotal"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_UrlChecked">
           <property name="text">
            <string>Checked</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_UrlChecked</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_UrlChecked"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_UrlDifferent">
           <property name="text">
            <string>Different</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_UrlDifferent</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_UrlDifferent"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_DNSStatus"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_DNSStatus">
           <property name="text">
            <string>DNS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_ViewOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>View Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_ViewShowAll">
           <property name="text">
            <string>Show all</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget_urls">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text">
          <string>URL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STATUS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CODE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>FORM</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TYPE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SUBTYPE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SIZE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PARENT</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
